numba/np/arraymath.py

```diff
@@ -3589,9 +3589,10 @@ def jit_np_intersect1d(ar1, ar2, assume_unique=False):
     # https://github.com/numpy/numpy/blob/v1.19.0/numpy/lib
     # /arraysetops.py#L347-L441
     if not (type_can_asarray(ar1) or type_can_asarray(ar2)):
-        raise TypingError('intersect1d: first two args must be array-like')
+        raise TypingError('np.intersect1d: first two args must be array-like')
```

Comment:
Although is it a common convention to import the NumPy package as `np`, it is not strictly true. I'd just keep the message as it was before, without the `np.` prefix.